src/DynamoCore/Graph/ModelBase.cs

```diff
@@ -173,21 +188,34 @@ public Guid GUID
             }
         }
 
+        /// <summary>
+        /// Protected constructor.
+        /// </summary>
         protected ModelBase()
         {
             GUID = Guid.NewGuid();
         }
 
+        /// <summary>
+        /// Selects object.
```

Comment:
Say - "this method sets the isSelected property on the object".
provide an example here. Like node.Select().